dev-embedded/ponyprog/Manifest

```diff
@@ -1 +1,2 @@
 DIST Pony_Prog2000-2.07c.tar.gz 653300 BLAKE2B 7b20450dfefb4370fb813b5532e6ba504b6ac8e598a2e46615ba522537e91b7798f6f86fbf93320fc71be5fe756ceef3351304196da191f7e68670f75ab5f865 SHA512 33cd14fa8283ce8af8265a48df7d960c7c2779551d0f204f6728a9e1e54204f52ea608c4e6090dc56f7634a02abd9bb960230f75a3a9716b1639708a57c31809
```

Comment:
But it's not a new package? Just a new version?

Reply:
new version. fixed